latch/types/metadata.py

```diff
@@ -393,6 +395,7 @@ class SnakemakeParameter(LatchParameter):
     default: Optional[Any] = None
 
 
+# DEPRECATED: use `file_metadata` keyword in `SnakemakeMetadata` instead
```

Comment:
find some way to do https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated

at the very least put this in the docstring